### DISCUSSION TITLE
邮箱配置

### DISCUSSION THREAD

**Discussion by HoanNam2011** [bs-community/blessing-skin-server#549] [Q&A] [answered]
我用的是腾讯企业邮
但是在验证过程中，皮肤站提示：
邮件发送失败，详细信息：Connection could not be established with host :stream_socket_client(): Unable to connect to :465 (由于目标计算机积极拒绝，无法连接。)
这怎么办，或许说是我的填写错误？

**Comment by CharCaius** [comment, marked as answer]
请检查您的皮肤站与腾讯企业邮的邮件服务器是否能够进行正常通信
